### ISSUE TITLE
stagehand web-angular not building

### ISSUE THREAD

**Issue by StorminGorman** [dart-lang/stagehand#606]
This is the total output from doing:
$ stagehand web-angular
$ pub get
$ webdev serve

➜  dart stagehand web-angular
Creating web-angular application `dart`:
  /Users/briangorman/Desktop/play/dart/.gitignore
  /Users/briangorman/Desktop/play/dart/CHANGELOG.md
  /Users/briangorman/Desktop/play/dart/README.md
  /Users/briangorman/Desktop/play/dart/analysis_options.yaml
  /Users/briangorman/Desktop/play/dart/lib/app_component.css
  /Users/briangorman/Desktop/play/dart/lib/app_component.dart
  /Users/briangorman/Desktop/play/dart/lib/app_component.html
  /Users/briangorman/Desktop/play/dart/lib/src/todo_list/todo_list_component.css
  /Users/briangorman/Desktop/play/dart/lib/src/todo_list/todo_list_component.dart
  /Users/briangorman/Desktop/play/dart/lib/src/todo_list/todo_list_component.html
  /Users/briangorman/Desktop/play/dart/lib/src/todo_list/todo_list_service.dart
  /Users/briangorman/Desktop/play/dart/pubspec.yaml
  /Users/briangorman/Desktop/play/dart/test/app_test.dart
  /Users/briangorman/Desktop/play/dart/web/favicon.png
  /Users/briangorman/Desktop/play/dart/web/index.html
  /Users/briangorman/Desktop/play/dart/web/main.dart
  /Users/briangorman/Desktop/play/dart/web/styles.css
17 files written.

--> to provision required packages, run 'pub get'
➜  dart pub get
Resolving dependencies... (7.8s)
+ analyzer 0.32.6 (0.36.3 available)
+ angular 5.0.0 (5.3.0 available)
+ angular_ast 0.5.6 (0.5.9 available)
+ angular_compiler 0.4.0 (0.4.3 available)
+ angular_components 0.9.2 (0.13.0 available)
+ angular_forms 2.0.0 (2.1.2 available)
+ angular_test 2.0.0 (2.3.0 available)
+ archive 2.0.8
+ args 1.5.1
+ async 2.2.0
+ bazel_worker 0.1.20
+ boolean_selector 1.0.4
+ build 0.12.8 (1.1.4 available)
+ build_config 0.3.2 (0.4.0 available)
+ build_modules 1.0.3 (2.1.2 available)
+ build_resolvers 0.2.2+7 (1.0.4 available)
+ build_runner 0.10.3 (1.4.0 available)
+ build_runner_core 1.0.0 (3.0.4 available)
+ build_test 0.10.6 (0.10.7+1 available)
+ build_web_compilers 0.4.4+3 (2.0.0 available)
+ built_collection 4.2.0
+ built_value 6.4.0
+ charcode 1.1.2
+ cli_repl 0.2.0+1
+ code_builder 3.2.0
+ collection 1.14.11
+ convert 2.1.1
+ crypto 2.0.6
+ csslib 0.14.6 (0.16.0 available)
+ dart2_constant 1.0.2+dart2
+ dart_internal 0.1.4
+ dart_style 1.1.3 (1.2.7 available)
+ fixnum 0.10.9
+ front_end 0.1.4+2 (0.1.18 available)
+ glob 1.1.7
+ graphs 0.1.3+1 (0.2.0 available)
+ html 0.14.0 (0.14.0+2 available)
+ http 0.12.0+2
+ http_multi_server 2.0.6
+ http_parser 3.1.3
+ intl 0.15.8
+ io 0.3.3
+ js 0.6.1+1
+ json_annotation 1.2.0 (2.2.0 available)
+ json_rpc_2 2.1.0
+ kernel 0.3.4+2 (0.3.18 available)
+ logging 0.11.3+2
+ matcher 0.12.3+1 (0.12.5 available)
+ meta 1.1.7
+ mime 0.9.6+2
+ multi_server_socket 1.0.2
+ node_preamble 1.4.4
+ observable 0.22.2
+ package_config 1.0.5
+ package_resolver 1.0.10
+ path 1.6.2
+ pedantic 1.5.0
+ plugin 0.2.0+3
+ pool 1.4.0
+ protobuf 0.10.8 (0.13.11 available)
+ pub_semver 1.4.2
+ pubspec_parse 0.1.4
+ quiver 0.29.0+2 (2.0.3 available)
+ sass 1.20.1
+ sass_builder 2.1.3
+ scratch_space 0.0.3+2
+ shelf 0.7.5
+ shelf_packages_handler 1.0.4
+ shelf_static 0.2.8
+ shelf_web_socket 0.2.3
+ source_gen 0.9.1+3 (0.9.4+2 available)
+ source_map_stack_trace 1.1.5
+ source_maps 0.10.8
+ source_span 1.5.5
+ stack_trace 1.9.3
+ stream_channel 1.7.0 (2.0.0 available)
+ stream_transform 0.0.19
+ string_scanner 1.0.4
+ term_glyph 1.1.0
+ test 1.5.3 (1.6.3 available)
+ test_api 0.2.2 (0.2.5 available)
+ test_core 0.2.1+1 (0.2.5 available)
+ timing 0.1.1+1
+ tuple 1.0.2
+ typed_data 1.1.6
+ vm_service_client 0.2.6+2
+ watcher 0.9.7+10
+ web_socket_channel 1.0.12
+ yaml 2.1.15
Downloading sass 1.20.1...
Changed 89 dependencies!
Precompiling executables... (16.9s)
Precompiled build_runner:graph_inspector.
Precompiled test:test.
Precompiled build_runner:build_runner.
➜  dart webdev serve
Creating build script, took 7869ms
[INFO] Setting up file watchers completed, took 23ms
[INFO] Waiting for all file watchers to be ready completed, took 205ms
[WARNING] Throwing away cached asset graph because the build phases have changed. This most commonly would happen as a result of adding a new dependency or updating your dependencies.
[INFO] Cleaning up outputs from previous builds. completed, took 11ms
[INFO] Reading cached asset graph completed, took 390ms
[INFO] Building new asset graph completed, took 1.5s
[INFO] Checking for unexpected pre-existing outputs. completed, took 0ms
[SEVERE] build_web_compilers:ddc on package:package_resolver/package_resolver.dartdevc.module:
Error compiling dartdevc module:package_resolver|lib/package_resolver.ddc.js

[error] The imported library ''current_isolate_resolver_stub.dart'' can't have a part-of directive. (package:package_resolver/src/package_resolver.dart, line 12, col 8)
[error] The imported library ''utils_isolate_stub.dart'' can't have a part-of directive. (package:package_resolver/src/utils.dart, line 15, col 8)

Please fix all errors before compiling (warnings are okay).
}
[WARNING] build_web_compilers:entrypoint on test/app_test.dart.browser_test.dart:
Unable to read package_resolver|lib/package_resolver.ddc.js, check your console or the `.dart_tool/build/generated/package_resolver/lib/package_resolver.ddc.js.errors` log file.
[INFO] Running build completed, took 1m 17s
[INFO] Caching finalized dependency graph completed, took 413ms
[SEVERE] Failed after 1m 17s
Serving `web` on http://localhost:8080
Serving `test` on http://localhost:8081

**Comment by kwalrath** [comment]
Things are a little crazy now with I/O coming up, but we should fix this ASAP.

**Comment by StorminGorman** [comment]
OK i think this has already been fixed.  I apologize.. I decided to take a look at the problem myself and even went through the process of signing the CLA and everything.  I got a better error once checking out the source and building from master... apparently I just needed to upgrade my webdev and stagehand.  

`webdev could not run for this project.
The `build_web_compilers` version – 1.2.1 – is not within the allowed constraint – >=0.3.6 <0.5.0.
`

After doing this I retried the `stagehand web-angular` and it worked..
